src/DevHome.csproj

```diff
@@ -71,13 +71,14 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.2428" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240311000" />

```

Comment:
I don't understand why most of the time, these PackageReferences will flow through ProjectReferences but sometimes they just don't. Do you need to include this here, or do we get it for free from importing DevHome.Common?

Reply:
Without this, nuget restore tries to find 1.3 WASDK for test projs. I think that's because they don't reference DevHome.Common and only reference DevHome?

Reply:
Should they reference DevHome.Common too?

Reply:
Oh that's weird. No, I'd rather have an extra reference to WASDK here than a bunch of references to DevHome.Common. My comment was trying to keep references down in general, since when lots of projects include the same package, it's easier for the versions to get out of sync.